build_runner/lib/src/generate/performance_tracker.dart

```diff
@@ -19,45 +19,82 @@ abstract class Timings {
 
 /// The [Timings] of an entire build, including all its [actions].
 abstract class BuildPerformance implements Timings {
-  /// The [Timings] of each [BuildAction] ran in this build.
-  Iterable<BuildActionPerformance> get actions;
+  /// The [Timings] of each phase ran in this build.
+  Iterable<BuildPhasePerformance> get phases;
```

Comment:
were you planning on adding an `@experimental` annotation to this library?

Reply:
done